### ISSUE TITLE
Erroneous Radio channel.

### ISSUE THREAD

**Issue by Rohaneanew** [HypatiaStation/Baystation12#29]
:h, and :c both go to the command channel, and is effectively redundant. Proposal: Remove :h.

**Comment by Dalekfodder** [comment]
I don't think this is a good idea. It's not a bug obviously. Some people are used to using the :h and some other :c
